### ISSUE TITLE
Reports are blank when path isn't specified

### ISSUE THREAD

**Issue by KellyDeveloped** [hrcorval/behavex#109]
**Describe the bug**
When running `behavex` without specifying a path, the HTML report generated shows no tests. It looks like the other reports generated don't have information either. The `report.json` file lists no features.

The tests are discovered, run, and show in stdout, but the features/scenario output seems to not be collected for the reports

The same thing happens when you provide a tag to be run.

**To Reproduce**
From the root of the project, run `behavex` without specifying any paths
Open the `output` folder and open any of the reports

**Expected behavior**
Running `behavex` should include all executed features in the report. 

**Screenshots**
Running `behavex` produces this output
![image](https://github.com/hrcorval/behavex/assets/11000575/5aef4732-5b6b-40d6-908e-775f24cb4d8a)

Which in turn produces this HTML report
![image](https://github.com/hrcorval/behavex/assets/11000575/993da5cf-77e2-4dec-beaa-96533cc6a365)

Running `behavex features` producecs the same output, with this additional line:
`HTML output report is located at: C:\Users\Liam\Documents\behave-playzone\output\report.html`

Which looks like this
![image](https://github.com/hrcorval/behavex/assets/11000575/c7e277d9-df91-412d-abab-39a173eb3264)

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 3.0.0

**Comment by KPOTOKPblC** [comment]
Hello!
I have the same problem. Are there any plans to fix it?

**Comment by hrcorval** [comment]
We cannot reproduce the issue above (we will continue trying to find a consistent way to reproduce it). However, we did lot of improvements in version 3.2.0 about feature processing, some maybe that was indirectly fixed. Can you please try with latest Behavex version? Thanks

**Comment by cosmocracy** [comment]
+1, getting empty reports despite the command-line runner showing successful execution of my feature/scenarios/steps. Almost moved on to another testing framework but found this issue (with the workaround to provide the `features` directive on the command-line, which worked like a charm--thank you @KellyDeveloped)

**Comment by cosmocracy** [comment]
Looks like there's a bug on line 125 of `runner.py`....
```python
if os.environ.get('FEATURES_PATH') == '':
```
Should instead be:
```python
if os.environ.get('FEATURES_PATH') is None or os.environ.get('FEATURES_PATH') == '':
```

**Comment by hrcorval** [comment]
Fixed in upcoming version. 
Related branch: https://github.com/hrcorval/behavex/compare/release_3.2.1

Thanks!

**Comment by hrcorval** [comment]
Released in BehaveX v3.2.12
